xml/System.Collections.Immutable/ImmutableDictionary`2.xml

```diff
@@ -2058,7 +2058,7 @@ This member is an explicit interface member implementation. It can be used only
       <Docs>
         <summary>Gets the value comparer used to determine whether values are equal.</summary>
         <value>The value comparer used to determine whether values are equal.</value>
-        <remarks>To be added.</remarks>
+        <remarks>Equal values do not get replaced in order to minimize the amount of copying required when updating an immutable collection.</remarks>
```

Comment:
I think slightly more context is needed here, see for example my previous comment https://github.com/dotnet/dotnet-api-docs/pull/7014#discussion_r685269452

Reply:
Done.